src/backend/core/tests/api/test_organizations_accounts.py

```
@@ -0,0 +1,448 @@
+##
+## Get account
+##
+
+@pytest.mark.parametrize("auth_method", ["user", "external_api_key"])
+def test_api_organizations_accounts_get(account_test_setup, auth_method):
+    """Operators using external API key should not be able to get accounts of other operators."""
+    client = APIClient()
+    if auth_method == "user":
+        client.force_login(account_test_setup['user'])
+    elif auth_method == "external_api_key":
+        client.credentials(HTTP_AUTHORIZATION=f"Bearer test-external-api-key-12345")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)